### ISSUE TITLE
Make the Signage Item to be displayed in the corridor panel, overriding defaults. Once its placed in the PETI TEST

### ISSUE THREAD

**Issue by ellingtonvengaboysmuseum** [TeamSpen210/BEE2-items#945]
It will need a override to remove the signage in the corridor when a signage item is placed in the test in PETI.

**Comment by TheDarkBomber** [comment]
We do not need to change the entry/exit door signage. Use the normal signage item.

**Comment by LautaroL20** [comment]
Yes, TheDarkBomber is right. So this should be an ehnancement to the current one, not a new item.

**Comment by ellingtonvengaboysmuseum** [comment]
closed, see updated idea about entry and exit corridor override

**Comment by LautaroL20** [comment]
This idea is focused in overriding default signages from the entry corridor´s panel. Your other post is mostly about the removal from entry / exit corridors, wich is pretty different.

**Comment by Luke18033** [comment]
Not sure what the suggestion is here, being able to customize chamber sign icons?

**Comment by LautaroL20** [comment]
Maybe he means that if the signage item was placed in the map, then he wants them to appear in the bottom of this panel: 

![Signage panel](https://user-images.githubusercontent.com/12244376/208025504-56840433-391a-4841-84dc-be1107402b09.jpg)
